runtimes/pythonrt/config.go

```diff
@@ -13,17 +17,22 @@ type Config struct {
 	LogBuildCode      bool `koanf:"log_build_code"`
 
 	RunnerType string `koanf:"runner_type"`
+
+	StartDuration time.Duration `koanf:"start_duration"`
```

Comment:
something like MAX_RUNNER_START_TIMOUET suits this use case more than start duration